test/controllers/EditSetControllerSpec.scala

```diff
@@ -1149,6 +1162,7 @@ class EditSetControllerSpec extends BaseSpec {
 
           }
           "doesn't exist" in {
+            pending
```

Comment:
I don't think the behaviour of this test is correct - I think that if a submit happens for a record that doesn't exist the response should be a bad request of some kind.

I'd like to get this checked and either delete this test or have confirmation of this behaviour before merging.

Reply:
@felixgb In future if the record doesn't exist then this would create a new record but currently we are only looking to edit existing records, not create new ones, so a bad request seems reasonable for now.